jina/types/score/map.py

```diff
@@ -0,0 +1,102 @@
+from typing import Optional, Union
+
+import numpy as np
+
+from ..score import NamedScore
+from ..mixin import ProtoTypeMixin
+from ...excepts import BadNamedScoreType
+from ...helper import typename
+from ...proto import jina_pb2
+
+__all__ = ['MappedNamedScore']
+
+
+class MappedNamedScore(ProtoTypeMixin):
```

Comment:
suggestion, refine the name to `NameScoreMapping`? Python community often use this kind of name  such as `Mapping`, `MutableMapping` etc..

Reply:
Good point!, Added